.github/workflows/manifold.yml

```diff
@@ -71,48 +71,47 @@ jobs:
         cd build/test
         node ./manifold_test.js
 
-  # unable to get it working for now
-  # build_windows:
-  #   strategy:
-  #     matrix:
-  #       backend: [CPP, CUDA]
-  #   runs-on: windows-2019
-  #   steps:
-  #   - uses: actions/checkout@v3
-  #     with:
-  #       submodules: true
-  #   - uses: jwlawson/actions-setup-cmake@v1.12
-  #   - name: Patch assimp
-  #     shell: bash
-  #     run: patch third_party/assimp/code/CMakeLists.txt < assimp.diff
-  #   - uses: ilammy/msvc-dev-cmd@v1
-  #   - name: Install CUDA
-  #     if: matrix.backend == 'CUDA'
-  #     env: 
-  #       cuda: "11.3.0"
-  #       visual_studio: "Visual Studio 16 2019"
-  #     shell: powershell
-  #     run: |
-  #       Invoke-RestMethod https://raw.githubusercontent.com/ptheywood/cuda-cmake-github-actions/21811d232fc775fa8d44dddf7f8c5ce2cf8d2b81/scripts/actions/install_cuda_windows.ps1 -OutFile install_cuda_windows.ps1
-  #       .\install_cuda_windows.ps1
-  #   - name: nvcc check
-  #     if: matrix.backend == 'CUDA'
-  #     shell: powershell
-  #     run: |
-  #       nvcc -V
-  #       ls $env:CUDA_PATH
-  #       ls $env:CUDA_PATH\bin
-  #       ls $env:CUDA_PATH\include
-  #   - name: Build ${{matrix.backend}}
-  #     shell: powershell
-  #     run: |
-  #       cmake . -DCMAKE_BUILD_TYPE=Release -B build -DTHRUST_BACKEND=${{matrix.backend}} -A x64 -G "Visual Studio 16 2019"
-  #       cd build
-  #       cmake --build .
-  #   - name: Test ${{matrix.backend}}
-  #     if: matrix.backend != 'CUDA'
-  #     shell: bash
-  #     run: |
-  #       cd build\test
-  #       .\manifold_test.exe
-  #
+  build_windows:
+    strategy:
+      matrix:
+        backend: [CPP, CUDA]
```

Comment:
Maybe do CUDA first so we can see if it builds before any CPP tests fail?

Reply:
It should now do CUDA first